samples/cpp/ParallelLoopBody.cpp

```diff
@@ -0,0 +1,123 @@
+#include <opencv2/core/core.hpp>
+#include <opencv2/imgproc/imgproc.hpp>
+#include "opencv2/highgui/highgui.hpp"
+#include <iostream>
+
+using namespace cv;
+using namespace std;
+
+// This class use ParallelLoopBody to process image using thread
+// ParallelMeanLine process image to have for each line a mean value of 128 and standard deviation of 64
+class ParallelMeanLine: public cv::ParallelLoopBody
```

Comment:
My objective is to give an ParrallelLoopBody example with a simple calculus (mean and variance line) and without any call to opencv functiong using ParallelLoopBody or thread.